openverse_catalog/dags/common/loader/reporting.py

```diff
@@ -15,9 +18,10 @@
     ("min", 60),
     ("sec", 1),
 )
+RecordCounts = dict[str, tuple[Optional[int], Optional[int]]]
```

Comment:
Ah, the `Optional` here is because we get `None` instead of `0` when 0 rows are returned for either `loaded` or `upserted`, right?

Reply:
That's right!